src/script/page/preferences/accountPreferences/HistoryBackupSection.tsx

```diff
@@ -24,57 +24,78 @@ import {HistoryExportViewModel} from '../../../view_model/content/HistoryExportV
 }
 
-const HistoryBackupSection: React.FC<HistoryBackupSectionProps> = ({brandName}) => (
-  <PreferencesSection
-    hasSeparator
-    title={t('preferencesOptionsBackupHeader')}
-    className="preferences-section-conversation-history"
-  >
-    <button
-      className="preferences-link accent-text"
-      onClick={() => {
-        {
-          amplify.publish(WebAppEvents.CONTENT.SWITCH, ContentViewModel.STATE.HISTORY_EXPORT);
-          amplify.publish(WebAppEvents.BACKUP.EXPORT.START);
-        }
-      }}
-      data-uie-name="do-backup-export"
-      aria-describedby="preferences-history-describe-1"
-      type="button"
-    >
-      {t('preferencesOptionsBackupExportHeadline')}
-    </button>
-    <p id="preferences-history-describe-1" className="preferences-detail">
-      {t('preferencesOptionsBackupExportSecondary', brandName)}
-    </p>
-    <label
-      className="preferences-link accent-text preferences-history-backup-import-field"
-      data-uie-name="do-backup-import"
+const HistoryBackupSection: React.FC<HistoryBackupSectionProps> = ({brandName}) => {
+  const fileInputRef = React.useRef(null);
+
+  const fileInputClick = () => {
+    fileInputRef.current.click();
+  };
+
+  return (
+    <PreferencesSection
+      hasSeparator
+      title={t('preferencesOptionsBackupHeader')}
+      className="preferences-section-conversation-history"
+      aria-label={t('preferencesOptionsBackupExportHeadline')}
     >
-      <span>{t('preferencesOptionsBackupImportHeadline')}</span>
-      <input
-        type="file"
-        accept={`.${HistoryExportViewModel.CONFIG.FILE_EXTENSION}`}
-        onChange={event => {
-          const file = event.target.files[0];
-          if (file) {
-            amplify.publish(WebAppEvents.CONTENT.SWITCH, ContentViewModel.STATE.HISTORY_IMPORT);
-            amplify.publish(WebAppEvents.BACKUP.IMPORT.START, file);
+      <button
+        className="preferences-link accent-text"
+        onClick={() => {
+          {
+            amplify.publish(WebAppEvents.CONTENT.SWITCH, ContentViewModel.STATE.HISTORY_EXPORT);
+            amplify.publish(WebAppEvents.BACKUP.EXPORT.START);
           }
         }}
-        data-uie-name="input-import-file"
-        aria-describedby="preferences-history-describe-2"
-      />
-    </label>
-    <p id="preferences-history-describe-2" className="preferences-detail">
-      {t('preferencesOptionsBackupImportSecondary')}
-    </p>
-  </PreferencesSection>
-);
+        data-uie-name="do-backup-export"
+        aria-describedby="preferences-history-describe-1"
+        type="button"
+      >
+        {t('preferencesOptionsBackupExportHeadline')}
+      </button>
+      <p id="preferences-history-describe-1" className="preferences-detail">
+        {t('preferencesOptionsBackupExportSecondary', brandName)}
+      </p>
+      <div
+        role="button"
+        tabIndex={0}
+        onKeyDown={(event: React.KeyboardEvent<HTMLElement>) => handleKeyDown(event, fileInputClick)}
```

Comment:
Here we probably also can add `aria-label`l or `aria-labelledby`

Reply:
done